packages/core/src/payment/strategies/googlepay/googlepay.ts

```diff
@@ -7,9 +8,33 @@ import {
     GooglePayBraintreeSDK,
 } from '../braintree';
 
+export enum CallbackTriggerType {
+    INITIALIZE = 'INITIALIZE',
+    SHIPPING_OPTION = 'SHIPPING_OPTION',
+    SHIPPING_ADDRESS = 'SHIPPING_ADDRESS',
+    OFFER = 'OFFER',
+}
+
+export enum TotalPriceStatusType {
+    ESTIMATED = 'ESTIMATED',
+    FINAL = 'FINAL',
+    NOT_CURRENTLY_KNOWN = 'NOT_CURRENTLY_KNOWN',
+}
+
+export enum CallbackIntentsType {
+    OFFER = 'OFFER',
+    PAYMENT_AUTHORIZATION = 'PAYMENT_AUTHORIZATION',
+    SHIPPING_ADDRESS = 'SHIPPING_ADDRESS',
+    SHIPPING_OPTION = 'SHIPPING_OPTION',
+}
+
 export type EnvironmentType = 'PRODUCTION' | 'TEST';
 export type TokenizeType = 'AndroidPayCard' | 'CreditCard' | 'CARD';
 
+export interface GooglePayBuyNowInitializeOptions {
+    getBuyNowCartRequestBody?(): BuyNowCartRequestBody | void;
```

Comment:
What are the scenario where the return type of this method might be a void?

Reply:
removed void